todo-backend/src/index.ts

```diff
@@ -0,0 +1,7 @@
+import app from "@server";
+
+// Start the server
+const port = Number(process.env.PORT || 8080);
```

Comment:
potential configmap variable?

Reply:
Yep, like above, as long as the environment variable gets set in the pod, the application should pick up the variable.